main.go

```diff
@@ -198,6 +199,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
 
+	fs.DurationVar(&nodeDrainClientTimeout, "node-drain-client-timeout-duration", time.Second*10,
+		"The timeout of the client used for draining nodes.")
```

Comment:
```suggestion
		"The timeout of the client used for draining nodes. Defaults to 10s if set to 0.")
```
last not from my side: just to surface the fact that we default. Makes it less surprising